caffe2/CMakeLists.txt

```diff
@@ -1697,8 +1697,9 @@ if(BUILD_PYTHON)
   # https://github.com/pytorch/pytorch/tree/master/tools/build_pytorch_libs.bat#note-backslash-munging-on-windows
   pycmd(PYTHON_SITE_PACKAGES "
       import os
-      from distutils import sysconfig
-      print(sysconfig.get_python_lib(prefix=''))
+      import sysconfig
+      relative_site_packages = sysconfig.get_path('purelib').replace(sysconfig.get_path('data'), '').lstrip(os.path.sep)
+      print(relative_site_packages)
```

Comment:
the heck is this

Reply:
`sysconfig` [hardcodes these paths with templates](https://github.com/python/cpython/blob/103d5e420dd90489933ad9da8bb1d6008773384d/Lib/sysconfig.py#L27-L38), but `distutils.sysconfig` doesn't and lets you supply your own prefix (i.e. `base` in the link), so here we end up having to remove the `base` (and its trailing slash) prefix manually.

```
>>> from distutils import sysconfig

>>> sysconfig.get_python_lib()
'/home/davidriazati/local/miniconda3/envs/pytorch/lib/python3.9/site-packages'

>>> sysconfig.get_python_lib(prefix='')
'lib/python3.9/site-packages'
```


```
>>> import sysconfig

>>> sysconfig.get_path('purelib')
'/home/davidriazati/local/miniconda3/envs/pytorch/lib/python3.9/site-packages'

>>> sysconfig.get_path('data') 
'/home/davidriazati/local/miniconda3/envs/pytorch'

```